Lib/idlelib/calltips.py

```diff
@@ -1,4 +1,4 @@
-"""calltips.py - An IDLE Extension to Jog Your Memory
+b"""calltips.py - An IDLE Extension to Jog Your Memory
```

Comment:
A typo..., I'm using trackpoint on laptop, sometime when scrolling down, it will unexpectedly
pressing "b"